edx_proctoring/migrations/0005_proctoredexam_hide_after_due.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('edx_proctoring', '0004_auto_20160201_0523'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='proctoredexam',
+            name='hide_after_due',
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
From my quick poke around the edxapp prod database, there are <200 rows in this table, so the migration should be very quick and not cause any performance issues.